src/gallery/_base.scss

```diff
@@ -1,97 +1,63 @@
+@import "https://cdnjs.cloudflare.com/ajax/libs/flickity/1.2.1/flickity.min.css";
+
```

Comment:
@misscs ah, good catch. I was thinking about pulling in the styles we want and adding it to the stylesheet, but for now I'll pull the file in locally and include. I think eventually we can restyle the Flickity controls ourselves, giving design an opportunity to think about them and just having our own CSS for it.